ern-orchestrator/src/codepush.ts

```diff
@@ -333,6 +333,14 @@ export async function performCodePushOtaUpdate(
       PackagePath.fromString(j.toString())
     )
 
+    const codePushConfig = await cauldron.getCodePushConfig(napDescriptor)
+    if (codePushConfig && codePushConfig.bypassYarnLock) {
+      pathToYarnLock = undefined
+      log.debug(
```

Comment:
repetitive , message from ern-local-cli/src/commands/create-container.ts will take care of it